bus-mapping/src/evm/opcodes.rs

```diff
@@ -522,6 +522,15 @@ pub fn gen_begin_tx_ops(
         // else, add 3 RW read operations for transaction L1 fee.
         gen_tx_l1_fee_ops(state, &mut exec_step);
     }
+
+    log::trace!("write tx l1fee {}", state.tx.l1_fee());
+    state.call_context_write(
```

Comment:
What about merging this `call_context_write` into the for loop below: https://github.com/scroll-tech/zkevm-circuits/blob/0bf839706606bcf30f75756b50b9253f329a081b/bus-mapping/src/evm/opcodes.rs#L544